README.md

```diff
@@ -321,9 +321,9 @@ The `<...>` notation means the argument.
   * Finish this frame. Resume the program until the current frame is finished.
 * `c[ontinue]`
   * Resume the program.
-* `q[uit]` or exit or `Ctrl-D`
+* `q[uit](!)` or `Ctrl-D`
```

Comment:
could you explain about `!` command?
I prefer to make separate lines like:

```
* q[uit] or ...
  * Finish ..
 * q[uit]!
   * Finish ...
```

Reply:
👍 updated